requirements-ci.txt

```diff
@@ -5,4 +5,5 @@ pytest-mock
 pytest-tornado
 mock
 docutils
+psycopg2-binary
```

Comment:
Why this change?

Reply:
Because of this https://github.com/julmon/temboard/blob/purge_data/temboardui/plugins/monitoring/__init__.py#L14